src/streaming/vg-dash/vg-dash.ts

```diff
@@ -48,7 +48,7 @@ export class VgDASH implements OnInit, OnChanges, OnDestroy {
         }
 
         // It's a DASH source
-        if (this.vgDash && this.vgDash.indexOf('.mpd') > -1) {
+        if (this.vgDash && ((this.vgDash.indexOf('.mpd') > 1) || (this.vgDash.indexOf('mpd-time-csf') > -1))) {
```

Comment:
Why `> 1`? It would be more correct `> -1` since it means that it actually found something. I know that this would work but `> -1` seems the proper way to handle this.